test/integration/state/initialState.t.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.8.23;
+
+import { IntegrationBaseSetup, IPowerToken, IZeroToken } from "../IntegrationBaseSetup.t.sol";
```

Comment:
import `IPowerToken` and `IZeroToken` from its origin location instead?

Reply:
Fixed the import paths in this commit: https://github.com/MZero-Labs/ttg/pull/263/commits/6cf80ea42e617e75069247b0dfbb9cfa871ecb80